contracts/foundry.toml

```diff
@@ -5,4 +5,22 @@ libs = ['lib']
 solc_version = '0.8.18'
 test = 'tests'
 
-# See more config options https://github.com/foundry-rs/foundry/tree/master/config
\ No newline at end of file
+[fmt]
+line_length = 100
+tab_width = 4
+bracket_spacing = false
+int_types = 'preserve'
+multiline_func_header = 'params_first'
+quote_style = 'double'
+number_underscores = 'preserve'
```

Comment:
I'd be down to change this to `'thousands'` based on the documentation here: https://book.getfoundry.sh/reference/config/formatter.